safe/definitionsv4/hazard_classifications.py

```diff
@@ -88,6 +85,73 @@
     ]
 }
 
+earthquake_mmi_hazard_classes = {
+    'key': 'earthquake_mmi_hazard_classes',
+    'name': tr('Earthquake MMI classes'),
+    'description': tr(
+        'Three classes are supported for volcano vector hazard data: '
```

Comment:
Volcano -> earthquake.

Wouldn't it be better to have one class for each MMI level?